src/components/site-search/components/SiteSearchForm.vue

```diff
@@ -116,13 +115,6 @@ export default {
             type: String,
             default: '',
         },
-        /**
-         * Text that renders inside the clear button (visually-hidden)
-         */
-        clearButtonText: {
```

Comment:
Removing a prop is a breaking change - Probably best to mark this as deprecated in the comment and create a separate branch to remove the prop, that branch can be merged to cl-release/v5.0.0 marked as a breaking change commit there